src/components/layouts/Dev.tsx

```diff
@@ -4,12 +4,15 @@ import { FacebookSessionInfo } from "../facebook/FacebookSessionInfo";
 import { TopNav } from "../nav/TopNav";
 import { SideNav } from "../nav/SideNav";
 
+import { Celebration } from "./Celebration";
+
 export const Dev = props => {
     const {facebookUserSession} = props;
     return <React.Fragment>
         <TopNav {...props} />
         <SideNav {...props} />
         <HomePage />
         <FacebookSessionInfo {...{facebookUserSession}} />
+        <Celebration numHeartpointsAwarded={10}/>
```

Comment:
so we basically need a way to decide whether or not to show the celebration popup, this should be passed in as props somehow, or influenced by the route! For state, what we do is we pass in a function from the top level that when called, would rerender the entire react site with an updated set of props, including a prop used in the decision criteria of whether or not to display the celebration! We can alternatively capture this in the URL, so only a certain URL would show the modal celebration, and then only when naving there do we see it, and naving away would cause it to go away! :)